index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Portfolio</title>
+  <link rel="stylesheet" href="style.css">
+</head>
+<body>
+  <div class="wrapper">
+    <header class="first">
+      <nav class="menu">
+        <ul>
+          <li><a href="#">Logo</a></li>
+          <li><a href="#"><img src="./images/union.svg" alt="Hamburger logo"></a></li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="second">
+      <h1>I'm Mariane <br>
```

Comment:
- [ ] Please make sure to use the same font and color as the Figma Template.

Your project
![image](https://user-images.githubusercontent.com/28109626/148312694-ba9a7cf6-cd4b-4cf1-b916-7239db7459fe.png)

Figma
![image](https://user-images.githubusercontent.com/28109626/148312717-cf56cd61-6718-4cca-96df-6e843f644410.png)

[Optional]

- [ ] Change the template name to your own name.